### DISCUSSION TITLE
Default for `max-doc-length` with `line-length` configured

### DISCUSSION THREAD

**Discussion by jamesbraza** [astral-sh/ruff#6459] [Q&A] [answered]
Today I learned of [`max-doc-length`](https://beta.ruff.rs/docs/settings/#pycodestyle-max-doc-length), its default value is `None`.  My question is, what does `None` mean?

Let's say I have `line-length = 97` (`ceil(1.1 * 88)`):
- Will `max-doc-length` implicitly be `97`?
- Or will `max-doc-length` be disabled entirely?

**Comment by zanieb** [comment, marked as answer]
@jamesbraza from some local testing it looks like the rule is disabled if the length is not specified

**Reply by zanieb** [reply]
Note this is also covered in the https://beta.ruff.rs/docs/rules/doc-line-too-long/ docs but I'll add a blurb to the setting in https://github.com/astral-sh/ruff/pull/6463

**Reply by jamesbraza** [reply]
Okay thank you!  I see that it's strictly opt-in now, thank you.

Can you perhaps add a link to [`doc-line-too-long` docs](https://beta.ruff.rs/docs/rules/doc-line-too-long/), in the comment(s) you're adding in #6463?
